### PR TITLE
ODS-5543 Claimset metadata IsEdfiPreset and ForApplicationUseOnly is included in the C#

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

**Comment by semalaiappan** [comment]
Claimset metadata has IsEdfiPreset and ForApplicationUseOnly now 
image

File: .github/workflows/edFi.ods.standard manual.yml [hunk 1]
```diff
@@ -34,12 +36,34 @@ jobs:
           repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
           path: Ed-Fi-ODS-Implementation/
           ref: main
+    - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
```
Comment (ericstockdale) [inline .github/workflows/edFi.ods.standard manual.yml:39]: @semalaiappan How are the changes in edfi.ods.standard manual.yml related to the ticket?

File: Application/EdFi.Ods.Api.IntegrationTestHarness/EdFi.Ods.Api.IntegrationTestHarness.csproj [hunk 2]
```diff
@@ -32,7 +32,7 @@
     <PackageReference Include="Autofac" Version="6.3.0" />
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="7.2.0" />
     <PackageReference Include="EdFi.Suite3.Admin.DataAccess" Version="6.0.12" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```
Comment (ericstockdale) [inline Application/EdFi.Ods.Api.IntegrationTestHarness/EdFi.Ods.Api.IntegrationTestHarness.csproj:35]: @semalaiappan I know this is a package that was already assigned and that you are updating the version, this package does not seem to be used by this project anywhere

File: Application/EdFi.Ods.SandboxAdmin/EdFi.Ods.SandboxAdmin.csproj [hunk 3]
```diff
@@ -17,7 +17,7 @@
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="7.2.0" />
     <PackageReference Include="EdFi.Suite3.Admin.DataAccess" Version="6.0.12" />
     <PackageReference Include="EdFi.Suite3.Common" Version="6.0.5" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```
Comment (ericstockdale) [inline Application/EdFi.Ods.SandboxAdmin/EdFi.Ods.SandboxAdmin.csproj:20]: @semalaiappan I know this is a package that was already assigned and that you are updating the version, this package does not seem to be used by this project anywhere

File: Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj [hunk 4]
```diff
@@ -23,7 +23,7 @@
 
   <ItemGroup>
     <PackageReference Include="EdFi.Suite3.Admin.DataAccess" Version="6.0.12" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```
Comment (ericstockdale) [inline Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj:26]: @semalaiappan I know this is a package that was already assigned and that you are updating the version, this package does not seem to be used by this project anywhere

File: .github/workflows/edFi.ods.standard manual.yml (hunk 1 above)
Comment (semalaiappan) [reply .github/workflows/edFi.ods.standard manual.yml]: Sorry . not related to this ticket and i removed it

File: Application/EdFi.Ods.Api.IntegrationTestHarness/EdFi.Ods.Api.IntegrationTestHarness.csproj (hunk 2 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.Api.IntegrationTestHarness/EdFi.Ods.Api.IntegrationTestHarness.csproj:35]: https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/ODS-5543/Application/EdFi.Ods.Api.IntegrationTestHarness/PostmanSecurityMetadataInitializer.cs#L7

File: Application/EdFi.Ods.SandboxAdmin/EdFi.Ods.SandboxAdmin.csproj (hunk 3 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.SandboxAdmin/EdFi.Ods.SandboxAdmin.csproj]: @ericstockdale removed it

File: Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj (hunk 4 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj]: @ericstockdale removed it

**Review by ericstockdale — APPROVED** [review]

**Review by ericstockdale — APPROVED** [review]

**Pull request merged**